CPAC/pipeline/engine.py

```diff
@@ -1937,7 +1953,17 @@ def ingress_raw_anat_data(wf, rpool, cfg, data_paths, unique_id, part_id, ses_id
             dl_dir=cfg.pipeline_setup["working_directory"]["path"],
             img_type="anat",
         )
-        rpool.set_data("T2w", anat_flow_T2, "outputspec.anat", {}, "", "anat_ingress")
+        reorient = pe.Node(
+            interface=afni.Resample(),
+            name=f"reorient_T1w_{part_id}_{ses_id}",
+        )
+
+        reorient.inputs.orientation = desired_orientation
+        reorient.inputs.outputtype = "NIFTI_GZ"
+
+        wf.connect(anat_flow_T2, "outputspec.anat", reorient, "in_file")
+
+        rpool.set_data("T2w", reorient, "out_file", {}, "", "anat_ingress")
```

Comment:
If we do this, then this can just be

```suggestion
        reorient = rpool.reorient_node("T2w", "anat_ingress", f"{part_id}_{ses_id}")
        wf.connect(anat_flow_T2, "outputspec.anat", reorient, "in_file")
```

Reply:
with the changes, I think this is not needed.